backend/src/packages/users/user.entity.ts

```diff
@@ -3,82 +3,119 @@ import { type Entity } from '#libs/types/types.js';
 class UserEntity implements Entity {
   private id: number | null;
 
+  private createdAt: Date | null;
+
+  private updatedAt: Date | null;
+
   private email: string;
 
+  private fullName: string;
+
   private passwordHash: string;
 
   private passwordSalt: string;
 
   private constructor({
     id,
     email,
+    fullName,
     passwordHash,
     passwordSalt,
+    createdAt,
+    updatedAt,
   }: {
     id: number | null;
     email: string;
+    fullName: string;
     passwordHash: string;
     passwordSalt: string;
+    createdAt: Date | null;
```

Comment:
We decided that constructor param with type string and filed with Date one in another PR)

Reply:
Yep, I am working on that at the moment, checked pr with sign In

Reply:
I see that you decided to use string in constructor here. But I wanted to ask, does find() function return object also with fullNam field? In my implementation findAll() returns an array with users+fullName field. [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/46#discussion_r1301798956)

Reply:
Or I think it's better to clarify it in issue